middleware/authorization-middleware.js

```diff
@@ -0,0 +1,38 @@
+var jwt = require("jsonwebtoken");
+
+const jwtTestingSign = "secret";
+
+const authorizationHeader = (allowedRole) => {
+  return (req, res, next) => {
+    const authHeader = req.headers.authorization;
+
+    if (!authHeader) {
+      res.status(401).json({
+        message: "Token is not available",
+      });
+    } else {
+      const token = authHeader.split(" ")[1];
```

Comment:
Add a line before variable declaration to make it more readable.

Reply:
> According to @suchipi , Github member:
> 
> _**adding and removing line breaks is core to prettier's formatting algorithm, so preserving them isn't consistently possible.**_

Preserving line breaks for this use case is impossible according to prettier github contributor. Probably as right now i still use prettier for easier style code and indent, But im still trying other code config like ESlint

Reply:
understandable